### DISCUSSION TITLE
Entities are no longer available !

### DISCUSSION THREAD

**Discussion by Veaullus** [pail23/stiebel_eltron_isg_component#185] [General]
Dear all,

Since January 11, several entities are no longer available:
- sensor.stiebel_eltron_isg_target_temperature_water
- sensor.stiebel_eltron_isg_outdoor_temperature
- sensor.stiebel_eltron_isg_actual_room_temperature_hk_1
- sensor.stiebel_eltron_isg_actual_temperature_water
...to name only the one that I already used almost daily, but there are others.

In my opinion, this is not a hardware or data availability problem, because it is available and correct via the platform SERVICEWELT.
Of course I restarted everything, whether it was my RPI (which hosts HA), the ISG module, or even the heat pump.
But nothing comes back.
I don't see a similar message here, so I assume I'm the only one experiencing the problem. I don't know if this is good news.

Do you have an idea ?

Thank you in advance, and have a good weekend ;-)

**Comment by Optic00** [comment]
Closing this as stale. There is no reproduction here and the Home Assistant and integration versions involved are long superseded. If entities still become unavailable on a current release, please open a new discussion with the controller model, integration version, and a debug log.
